pkg/app.go

```diff
@@ -192,7 +192,7 @@
 
 	wg.Wait()
 
-	queriesMetrics := NewQueriesMetrics(a.Config, queryInfos)
+	queriesMetrics := NewQueriesMetrics(a.Config.Chains, queryInfos)
```

Comment:
Do you have any learnings on app.go file regarding that?

Reply:
Hm, I remember telling that `pkg/app.go` isn't tested yet and for now this okay, and it should be in one of the learnings. So that means you don't have access to it, right?

Reply:
I see the following learning in your database:

> User freak12techno has clarified that while specific tests for new generator functions in `app.go` exist, there is an overall lack of tests for `app.go` itself, which is currently expected and accepted.

created on `June 25th 2024, 10:32:53 pm`. Do you think it's out of context, or do you not have any access to verify if it's related?

Reply:
Is there a way to interact with you so I can say once "pkg/app.go has no tests as of now and it's acceptable for now" and such tips won't be appearing in the future PRs?

Reply:
Can I configure it by communicating it with you in the comments somehow, just like I do now?

Reply:
Can I provide a feedback about the usage of the app to its developers via comments?